File: 0349-intersection-of-two-arrays/0349-intersection-of-two-arrays.cpp
class Solution {
public:
    vector<int> intersection(vector<int>& nums1, vector<int>& nums2) {
        vector<int>ans;
        for(int i =0;i<nums1.size();i++){
            for(int j =0;j<nums2.size();j++){
                if(nums1[i] == nums2[j])
                 ans.push_back(nums1[i]);
            }
        }
        set<int> s(ans.begin(),ans.end());
        return vector<int> (s.begin(),s.end());
    }
};

File: 0349-intersection-of-two-arrays/0349-intersection-of-two-arrays.py
class Solution(object):
    def intersection(self, nums1, nums2):
       nums1.sort()
       nums2.sort()
       s=set()
       for i in range(len(nums1)):
            for j in range(len(nums2)):
                if nums1[i]==nums2[j]:
                    s.add(nums1[i])
                    
       return list(s)
        